packages/client/src/views/SysAdmin/SysAdminContentWrapper.tsx

```diff
@@ -13,7 +13,7 @@ import { MarginedHeader } from '@client/components/Header/Header'
 import { CircleButton } from '@opencrvs/components/lib/buttons'
 import { BackArrowDeepBlue } from '@opencrvs/components/lib/icons'
 import { BodyContent, Container } from '@opencrvs/components/lib/Content'
-import { LoadingGrey } from '@opencrvs/components/lib/ListTable'
+import { LoadingGrey } from '@opencrvs/components/src/LoadingGrey'
```

Comment:
We should import directly from `@opencrvs/components` (you might need to run `yarn recreate-src-index-ts` in components & `yarn build`). The src approach works within OpenCRVS but if someone was to separately install the component library, the source files might not exist.

ESLint rule or something within TypeScript would be fantastic to disallow this, I know it's often the auto-import that does it :/